model_compression_toolkit/core/common/mixed_precision/mixed_precision_search_manager.py

```diff
@@ -212,7 +219,10 @@ def compute_node_ru_for_candidate(self, conf_node_idx: int, candidate_idx: int,
 
         """
         cfg = self.replace_config_in_index(self.min_ru_config, conf_node_idx, candidate_idx)
-        return self.compute_ru_functions[target].metric_fn(cfg, self.graph, self.fw_info, self.fw_impl)
+        if target == RUTarget.ACTIVATION:
```

Comment:
it's duplicated with the code under  compute_min_ru just the cfg is different. maybe extract this to a class protected function?